aikau/src/test/resources/alfresco/lists/AlfSortablePaginatedListTest.js

```diff
@@ -58,80 +99,141 @@ define(["intern!object",
       },
 
       "Count the rows": function() {
-         return browser.findAllByCssSelector(".alfresco-lists-views-layouts-Row")
+         return browser.findAllByCssSelector("#HASH_LIST .alfresco-lists-views-layouts-Row")
             .then(function(elements) {
                assert.lengthOf(elements, 20, "There should only be twenty rows");
             });
       },
 
-      "Post Coverage Results": function() {
-         TestCommon.alfPostCoverageResults(this, browser);
-      }
-   });
-
-   registerSuite({
-      name: "AlfSortablePaginatedList Tests (data load failure)",
-      
-      setup: function() {
-         browser = this.remote;
-         return TestCommon.loadTestWebScript(this.remote, "/AlfSortablePaginatedListDataFail", "AlfSortablePaginatedList Tests (data load failure)").end();
-      },
-      
-      beforeEach: function() {
-         browser.end();
-      },
-
-      "Check data failure message": function() {
-         return browser.findByCssSelector("#LIST .data-failure")
-            .isDisplayed()
-            .then(function(displayed) {
-               assert.isTrue(displayed, "Loading message not displayed");
+      "Scroll to bottom of basic infinite scroll area": function() {
+         // Click on the first row to give it focus...
+         return browser.findByCssSelector("#INFITE_SCROLL_LIST tr:nth-child(1) .alfresco-renderers-Property")
+            .click()
+            .pressKeys(keys.ARROW_DOWN)
+            .pressKeys(keys.ARROW_DOWN)
+            .pressKeys(keys.ARROW_DOWN)
+            .pressKeys(keys.ARROW_DOWN)
+            .pressKeys(keys.ARROW_DOWN)
+            .pressKeys(keys.ARROW_DOWN)
+            .pressKeys(keys.ARROW_DOWN)
+            .pressKeys(keys.ARROW_DOWN)
+         .end()
+         .getLastPublish("INFINITE_SCROLL_AREA_ALF_EVENTS_SCROLL")
+         .end()
+         .findAllByCssSelector("#INFITE_SCROLL_LIST tr")
+            .then(function(elements) {
+               assert.lengthOf(elements, 20, "Additional rows were not loaded when the bottom of the list was reached");
             });
       },
 
-      "Check that the pagination controls are all hidden": function() {
-         return browser.findByCssSelector("#PAGINATOR_PAGE_SELECTOR")
-            .isDisplayed()
-            .then(function(displayed) {
-               assert.isFalse(displayed, "The page selector was NOT hidden");
-            })
+      "Simulate a filter data request": function() {
+         // Clear previous pub/sub log (so that we can detect the next load)...
+         return browser.findByCssSelector(".alfresco_logging_DebugLog__clear-button")
```

Comment:
[My latest PR](http://www.watchthisspace.com) actually modifies TestCommon to have a clearLog() function ... will be worth updating this at some point, once it's been merged